lib/datasource/docker/common.ts

```diff
@@ -81,12 +81,20 @@ export async function getAuthHeaders(
         'base64'
       );
       opts.headers = { authorization: `Basic ${auth}` };
+    } else if (opts.token) {
+      const authType = opts.authType ? opts.authType : 'Bearer';
```

Comment:
```suggestion
      const authType = opts.authType ?? 'Bearer';
```

Simplify 🙃

Reply:
Yes, like I said: my first code in this specific language.